Makefile

```diff
@@ -1,15 +1,19 @@
-.PHONY: test
-test: node_modules
+.PHONY: test install
+
+export FOUNDRY_TEST=test-foundry
+
+install:
+	@git submodule update --init --recursive
+	@yarn
+
+test:
 	@echo Run all tests
-	@forge test -vvv -c test-foundry
+	@forge test -vvv
```

Comment:
I think that for these simple repos, we can do without makefile (only foundry.toml). People that are used to foundry will prefer

Reply:
Sure, but it does not cost much to also have the makefile. Plus it is really standard to have a makefile for different projects (not necessarily even related to blockchain)

Reply:
Ok! Let's keep it